db/migrations/002.do.active-deals.sql

```diff
@@ -7,7 +7,7 @@ CREATE TABLE active_deals (
   term_start_epoch INT NOT NULL,
   term_min INT NOT NULL,
   term_max INT NOT NULL,
-  sector_id INT NOT NULL,
+  sector_id BIGINT NOT NULL,
```

Comment:
You cannot modify existing columns this way, you need to create a new migration script that will `ALTER TABLE active_deals`